File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    player1 = []
    player2 = []
    for i in range(1,len(food)):
        for j in range(food[i] // 2):
            player1.append(str(i))
    for j in range(len(food)-1,0,-1):
        for i in range(food[j] // 2):
            player2.append(str(j))
    answer = player1 + [str(0)] + player2
    return ''.join(answer)

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    let temp ="";
    for(let i=1; i<food.length;i++){
        if(food[i] <2){
            continue;
        }
        else{
            if(food[i] % 2 ==0){
                for(let j=0; j<food[i] /2 ; j++){
                    temp += String(i);
                }
            }
            else{
                for(let j=0; j<(food[i] -1) / 2; j++){
                    temp += String(i);
                }
            }
        }
    }
    
    let reverse = temp.split("").reverse().join("");
    return temp + "0" + reverse;
}